### ISSUE TITLE
Allow freeze of last+first n columns?

### ISSUE THREAD

**Issue by a-schild** [FlowingCode/GridHelpers#29]
Hello,

it would be nice, if we could freeze the last n columns, so they always remain visible.
The idea is the same as on the selection column, but to be able to do this for the n columns at the start and/or end of the grid.

In our usecase we have a column with row "actions" (Delete/Edit etc.) on the last column.
When the columns are to large, then the action buttons are not visible any longer.
So it would be nice to have these columns always visible.

Thanks for considering this, not sure if it's doable in this project or if the basic grid would need more functionality for this

**Comment by javier-godoy** [comment]
Hello. That feature is implemented in Vaadin 23.1. You need to call [`setFrozenToEnd`](https://vaadin.com/api/platform/23.1.0/com/vaadin/flow/component/grid/Grid.Column.html#setFrozenToEnd(boolean)) for the rightmost columns (#https://github.com/vaadin/flow-components/issues/2859).

```
    grid.getColumns().get(0).setFrozen(true);
    grid.getColumns().get(9).setFrozenToEnd(true);
```

**Comment by a-schild** [comment]
Ah, thanks
